features/filesystem/sd/SDBlockDevice.h

```diff
@@ -127,7 +127,7 @@ class SDBlockDevice : public BlockDevice {
      */
     virtual void debug(bool dbg);
 
-private:
+protected:
```

Comment:
> As a temporary solution we can just change the access to protected (accept the pr), so that the iSDIO and other libraries can adopt the library, and move them back to private access when we have replacement functions.

that would be breaking , wouldn't be? If yes, I would prefer to go with the first one